PROCESSORS.md

```diff
@@ -2862,6 +2863,44 @@ In the list below, the names of required properties appear in bold. Any other pr
+| **Line Split Count**          |               |                  | The number of lines that will be added to each split file, excluding header lines. A value of zero requires Maximum Fragment Size to be set, and line count will not be considered in determining splits.                                                                                                                                                                                                                                                                                             |
+| Maximum Fragment Size         |               |                  | The maximum size of each split file, including header lines. NOTE: in the case where a single line exceeds this property (including headers, if applicable), that line will be output in a split of its own which exceeds this Maximum Fragment Size setting.                                                                                                                                                                                                                                         |
+| **Header Line Count**         | 0             |                  | The number of lines that should be considered part of the header; the header lines will be duplicated to all split files.                                                                                                                                                                                                                                                                                                                                                                             |
+| Header Line Marker Characters |               |                  | The first character(s) on the line of the datafile which signifies a header line. This value is ignored when Header Line Count is non-zero. The first line not containing the Header Line Marker Characters and all subsequent lines are considered non-header                                                                                                                                                                                                                                        |
+| **Remove Trailing Newlines**  | true          | true<br/>false   | Whether to remove newlines at the end of each split file. This should be false if you intend to merge the split files later. If this is set to 'true' and a FlowFile is generated that contains only 'empty lines' (i.e., consists only of and characters), the FlowFile will not be emitted. Note, however, that if header lines are specified, the resultant FlowFile will never be empty as it will consist of the header lines, so a FlowFile may be emitted that contains only the header lines. |
```

Comment:
This is a typo in NiFi, but we don't have to be typo-for-typo compatible :)
```suggestion
| **Remove Trailing Newlines**  | true          | true<br/>false   | Whether to remove newlines at the end of each split file. This should be false if you intend to merge the split files later. If this is set to 'true' and a FlowFile is generated that contains only 'empty lines' (i.e., consists only of \r and \n characters), the FlowFile will not be emitted. Note, however, that if header lines are specified, the resultant FlowFile will never be empty as it will consist of the header lines, so a FlowFile may be emitted that contains only the header lines. |
```

Reply:
Good catch, updated in fccf152353ae7c25a8d4944b8b0e3add15254984